general/solomiia_tymoshchuk/Task8.java

```diff
@@ -0,0 +1,46 @@
+package general.solomiia_tymoshchuk;
+
+/**
+ * Write a program that prints all numbers from 1 to N, that are divisible by 2, 3 or by both.
+ * If a number is divisible by only 2 - print it and add in parentheses “(by 2)”,
+ * if only by 3 - “(by 3)”, if by both numbers - “(by 2 and 3)”.
+ * Every number should be printed on the new line.
+ * Create two versions of the program: one should use the for loop, second - the while loop.
+ **/
+public class Task8 extends Task7 {
```

Comment:
Class `Task8` is not a class `Task7`, so it's better to not do this. If you want to have some utility methods, it's better to create a new class with utility functionality and use it wherever you need it.